src/mmw/apps/modeling/models.py

```diff
@@ -131,6 +161,20 @@ class Meta:
         auto_now_add=True)
     modified_at = models.DateTimeField(
         auto_now=True)
+    weather_type = models.CharField(
+        max_length=255,
+        default=WeatherType.DEFAULT,
+        choices=WeatherType.choices,
+        help_text='The source of weather data for this scenario. '
+                  'Only applies to GWLF-E scenarios.')
+    weather_simulation = models.CharField(
```

Comment:
Even if the weather data are saved as CSVs on S3, this will not be a file field. This field needs to be an identifier which can be used to look up the source of the simulated data for use, this character field seemed most appropriate.

As to where the data will actually be stored, the two options were in a table in the database, or as a dictionary in Python. Database fields can be changed without doing app releases, so adding more simulations in the future can be done unencumbered with releasing. However, having it as a dictionary in Python allows for stronger vetting and safety, as field will be tested in different environments. And it would be easier, not requiring further `setupdb` scripts. Thus, I lean towards the Python dictionary.

Implicit in this current design is the idea that either `weather_simulation` holds a direct reference to the source of the data (such as an S3 path), or holds a reference to an identifier used to look that up (the key of a dictionary, whose values are S3 paths). I'm leaning more towards the latter, but this field should suffice in either case.

However, I'm open to changing this field if we learn new information down the line. Right now it is mainly a justification for having `SIMULATION` as a choice for `weather_type`.